ext/web/03_abort_signal.js

```diff
@@ -34,6 +35,24 @@
       return signal;
     }
 
+    static timeout(millis) {
+      const prefix = "Failed to call 'AbortSignal.timeout'";
+      webidl.requiredArguments(arguments.length, 1, { prefix });
+      millis = webidl.converters["unsigned long long"]({ enforceRange: true });
+
+      const signal = new AbortSignal(illegalConstructorKey);
+      // TODO(andreubotella): Don't block the event loop from exiting if there
+      // aren't any listeners for the abort event.
```

Comment:
Should be fairly simple with `Deno.unrefTimer` & `Deno.refTimer`, though they are unstable

Reply:
I wasn't sure about overriding `addEventListener` and `removeEventListener` though, with such short timeframe before the release of 1.19.